packages/core/src/render3/error_code.ts

```diff
@@ -41,5 +41,15 @@ export class RuntimeError extends Error {
 /** Called to format a runtime error */
 export function formatRuntimeError(code: RuntimeErrorCode, message: string): string {
   const fullCode = code ? `NG0${code}: ` : '';
+
+  const HAS_GUIDE = new Set([
```

Comment:
May be also add a quick comment above the `HAS_GUIDE` const to describe what it is:
```suggestion
  // Contains a set of error messages that have details guides at angular.io.
  // Full list of available error guides can be found at https://angular.io/...
  const HAS_GUIDE = new Set([
```